lib/access_import/accessors/personen_rolle.rb

```diff
@@ -21,26 +21,26 @@ def map_rolle(z_rolle)
   end
 
   def all_volunteers
-    @records.select do |_id, val|
-      val[:z_Rolle] == 1
+    all.select do |_id, personen_rolle|
+      personen_rolle[:z_Rolle] == 1
```

Comment:
Yes sorry, too much JavaScript lately :smile:.

But you could use [OpenStruct](http://ruby-doc.org/stdlib-2.4.1/libdoc/ostruct/rdoc/OpenStruct.html) to achieve this syntax, but it's probably overkill.